packages/nameguard-react/src/utils/text.ts

```diff
@@ -2,7 +2,7 @@ import { CheckResultCode, Rating } from "@namehash/nameguard";
 
 export function checkResultCodeTextColor(
   check: CheckResultCode,
-  isHover = false
+  isHover = false,
 ): string {
   let color = "";
```

Comment:
Better to just make this `let color: string | null = null;` or something like that.